Source/LinqToDB/DataContext.cs

```diff
@@ -45,7 +45,7 @@ public DataContext() : this(DataConnection.DefaultDataOptions)
 		public DataContext(string? configurationString)
 			: this(configurationString == null
 				? DataConnection.DefaultDataOptions
-				: DataConnection.ConnectionOptionsByConfigurationString.GetOrAdd(configurationString, _ => new(new(configurationString))))
+				: DataConnection.ConnectionOptionsByConfigurationString.GetOrAdd(configurationString, _ => new(new(ConfigurationString : configurationString))))
```

Comment:
Nit: Remove space - does not match C# standards for named parameters. Same for other instances.

```suggestion
				: DataConnection.ConnectionOptionsByConfigurationString.GetOrAdd(configurationString, _ => new(new(ConfigurationString: configurationString))))
```